src/main/resources/assets/react/utils/mustStartWithALowercaseLetter.mjs

```diff
@@ -0,0 +1,9 @@
+const STARTS_WITH_ANYTHING_BUT_LOWERCASE_LETTER_REGEXP = /^[^a-z]/;
+
+
+export function mustStartWithALowercaseLetter(v) {
```

Comment:
Its a bit strange that checking methods return strings. 
Could you change this to something different?